### DISCUSSION TITLE
Allow live timing windows to be zoomed independently

### DISCUSSION THREAD

**Discussion by Petarkco** [f1multiviewer/issue-tracker#187] [Feature requests]
This is on a 27" 1440p screen:
![image](https://user-images.githubusercontent.com/25685344/188264157-1b99fc83-62f8-4ed0-b8b6-369fb0034c2b.png)

Currently the only fix is to zoom in but that effects all elements in every window.

**Comment by jeppeolesen** [comment]
Editing this to serve as a combination of this, #583 and #577.

User is requesting that individual live timing windows can be zoomed independently of the rest of the MultiViewer app.

**Comment by jeppeolesen** [comment]
Actually, we'll merge this into #194 as well, since that has more votes.
